src/www/app/electron_main.ts

```diff
@@ -111,7 +110,7 @@ main({
   getErrorReporter: (env: EnvironmentVariables) => {
     // Initialise error reporting in the main process.
     ipcRenderer.send('environment-info', {'appVersion': env.APP_VERSION, 'dsn': env.SENTRY_DSN});
-    return new ElectronErrorReporter(env.APP_VERSION, env.SENTRY_DSN || '');
+    return new ElectronErrorReporter(env.APP_VERSION, env.SENTRY_DSN || '', new URL(document.URL).searchParams.get('appName'));
```

Comment:
Do we need `appName || ''` (or some hardcoded value) to be safe?

Reply:
It's filled in by Electron from package.json, so it should always exist.